src/common/baseTelemetryReporter.ts

```diff
@@ -148,13 +148,16 @@ export class BaseTelemetryReporter {
 	// __GDPR__COMMON__ "common.remotename" : { "classification": "SystemMetaData", "purpose": "FeatureInsight" }
 	// __GDPR__COMMON__ "common.isnewappinstall" : { "classification": "SystemMetaData", "purpose": "FeatureInsight" }
 	// __GDPR__COMMON__ "common.product" : { "classification": "SystemMetaData", "purpose": "FeatureInsight" }
+	// __GDPR__COMMON__ "common.telemetryclientversion" : { "classification": "SystemMetaData", "purpose": "FeatureInsight" }
 	private getCommonProperties(): TelemetryEventProperties {
 		const commonProperties: Record<string, any> = {};
 		commonProperties["common.os"] = this.osShim.platform;
 		commonProperties["common.nodeArch"] = this.osShim.architecture;
 		commonProperties["common.platformversion"] = (this.osShim.release || "").replace(/^(\d+)(\.\d+)?(\.\d+)?(.*)/, "$1$2$3");
 		commonProperties["common.extname"] = this.extensionId;
 		commonProperties["common.extversion"] = this.extensionVersion;
+		// TODO @lramos15, this should be dynamic but loading JSON modules didn't play nicely
+		commonProperties["common.telemetryclientversion"] = "0.6.3";
```

Comment:
@connor4312 Any ideas on how to import a `JSON` file across node12, node 14, and node16. Seems `node 16` gets upset that I don't use import assertions but then `node12` and `node14` get upset when I do. I don't really want to rely on require because I use [no var requires](https://typescript-eslint.io/rules/no-var-requires/) and `fs` wouldn't work in a common layer

Reply:
Prior to Node 16, JSON modules [required a flag](https://github.com/nodejs/node/pull/41736) to be used in an ES6 import, so this is tricky.

Can the version be passed in from a higher level outside of the common layer?

Reply:
I could pass it in from the browser layer and node layer respectively. Not sure how that helps especially in the browser case. Normally I would use something like the bundler to compile time inject the value here but the idea is to avoid bundlers

Reply:
Honestly the easiest way would probably be just having a npm `postversion` script that would update the version number in a ts file when the version gets bumped